app/units/page.jsx

```diff
@@ -1,109 +1,132 @@
+"use client";
+import React, { useState } from 'react';
 import "./units.css";
 
 function Units() {
+    const [isFilterVisible, setIsFilterVisible] = useState(false);
+
+    const toggleFilter = () => {
+        setIsFilterVisible(!isFilterVisible);
+    };
+
+    const handleInputChange = () => {
+        setIsFilterVisible(false);
+    };
+
+    const clearForm = () => {
+        document.querySelector(".searchFilter").reset();
```

Comment:
Instead of using reset. Conditionally add CSS based on the state, that changes the height of the accordion to 0, and then add a transition with this to make it look nice.  Perhaps also apply this to the colour change implemented #31. 

Relevant Links:
- [Conditional CSS](https://stackoverflow.com/questions/30533171/conditionally-applying-class-attributes-in-react).
- [Transition](https://developer.mozilla.org/en-US/docs/Web/CSS/transition).

Reply:
Do you mean add the transition for the colour change?

Reply:
Mb, I wanted a transition, but we can do that when we move to shadow UI. Perhaps could you create an issue to migrate?